### PR TITLE
Increase gunicorn timeout and add expanded_menu for brb eateries

### PR DESCRIPTION
**Pull request by ilikesymmetry**

### CODE REVIEW

File: src/eatery/common_eatery.py [hunk 1]
```diff
@@ -101,6 +108,82 @@ def parse_events(event_list, event_date):
...
+
+  Returns a list of DescriptiveFoodStationTypes available to a certain category of an eatery.
+
+  Args:
+    eatery_stations (dict): A valid json segment from the hard-coded menu items
+  """
+  stations = []
+  for station in eatery_stations:
+    expanded_station = DescriptiveFoodStationType(
+      category=station['station'],
```
Comment (kq-li) [inline src/eatery/common_eatery.py:144]: this should be an extra indent (for your next pr)

File: src/eatery/static_eatery.py [hunk 2]
```diff
@@ -11,14 +11,15 @@
     parse_dining_items,
     parse_eatery_type,
     parse_events,
+    parse_expanded_menu,
     parse_payment_methods,
     parse_payment_methods_enum,
     resolve_id,
-    today,
+    today
```
Comment (kq-li) [inline src/eatery/static_eatery.py:18]: keep this trailing comma

File: src/eatery/static_eatery.py [hunk 3]
```diff
@@ -11,14 +11,15 @@
...
 )
 from src.types import (
     CampusEateryType,
-    OperatingHoursType,
+    OperatingHoursType
```
Comment (kq-li) [inline src/eatery/static_eatery.py:22]: and here

**Review by kq-li — APPROVED** [review]
lgtm, just a couple style things for next pr that aren't really worth opening issues for

**Review by orkosinha — APPROVED** [review]
lgtm!

**Pull request merged**